RocketChat/Rocket.Chat
# fix: External avatar url provider for rooms not respected.

## Proposed changes (including videos or screenshots)





## Issue(s)

Closes #28351

## Steps to test or reproduce

- Get a link to an image
- go to `Admin -> Settings -> Accounts -> Avatar -> Room Avatar External Provider URL`
- paste the link there
- save

All rooms should have this link as avatar.
## Further comments

[TC-635]

[TC-635]: https://rocketchat.atlassian.net/browse/TC-635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
